src/Abp/MimeTypes/MimeTypeMap.cs

```diff
@@ -0,0 +1,858 @@
+﻿// Taken from https://github.com/samuelneff/MimeTypeMap
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Abp.MimeTypes
+{
+    public class MimeTypeMap : IMimeTypeMap
+    {
+        protected const string Dot = ".";
+        protected const string QuestionMark = "?";
+
+        protected readonly Lazy<IDictionary<string, string>> MappingDictionary;
+
+        public MimeTypeMap()
+        {
+            MappingDictionary = new Lazy<IDictionary<string, string>>(BuildMappings);
```

Comment:
It would be great if we allow to change or extend the built-in **MappingDictionary **.